site/_config.yml

```diff
@@ -157,7 +163,8 @@ plugins:
   - jekyll-optional-front-matter # Parse Markdown files that do not have front-matter callouts
   - jekyll-titles-from-headings # pull the page title from the first Markdown heading when none is specified.
   - jekyll-paginate # pagination object for collections (e.g. posts)
-
+  - jekyll-redirect-from
+#  - data_page_generator
```

Comment:
curious why we have to include the code for this one vs. referencing it as a plugin? If that's necessary, let's delete this commented out . line

Reply:
It doesn't appear to be published on Ruby gems, unlike the others.

Reply:
Yeah it's a manual add, I've now removed the commented out line.